students/Chris_Kenyon/Lesson_3/list_lab.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# Lesson_3 Activity 2 List Lab
+
+# Series 1
+series_1 = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(series_1)
+
+# get user input for new fruit and add it to the end of the list
+add_fruit = input('Enter an additional fruit name: ')
+series_1.append(add_fruit)
+print(series_1)
+
+# get a number from user and diplay that number and the fruit in that position
+while True:
+    try:
+        user_num = int(input('Pick a fruit in the list between 1 and {}: '
+                       .format(len(series_1))))
+    except ValueError:
+        print("you have made an invalid choice, try again.")
+        continue
+    if user_num > 0 and user_num < len(series_1)+1:
+        print('The fruit that is number {} in the list is {}'
+              .format(user_num, series_1[user_num-1]))
+        break
+    print("you have made an invalid choice, try again.")
+
+# add string to font of list using +
+add_fruit_2 = input('Enter a fruit to add to the front of the list: ')
+series_1 = [add_fruit_2] + series_1
+print(series_1)
+
+# add string to fruont of list using insert
+add_fruit_3 = input("Enter another fruit to add"
+                    " to the front of the list: ")
+series_1.insert(0, add_fruit_3)
+print(series_1)
+
+# display the the fruits that begin with the letter p
+p_list = []
+for i in series_1:
+    if str.lower(i[0]) == 'p':
+        p_list.append(i)
+print("The fruits in this list that start with p are: \n{}".format(p_list))
+
+
+# Series 2
+print("---------------------------------")
+
+# print the series, remove the last item in the list, and then print again
+series_2 = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(series_2)
+series_2.pop()
+print(series_2)
+
+# multiply series by 2
+series_2 = series_2*2
+print("The fruits had babies \n {}".format(series_2))
+
+# While loop to check for proper input and remove fruit from list
+while True:
+    rem_fruit = input('Please select a fruit from the list to remove: ')
+    if rem_fruit in series_2:
+        while rem_fruit in series_2:
+            series_2.remove(rem_fruit)
+        break
+    print("That fruit is not on the list, please"
+          " check spelling and capitalization and try again")
+print(series_2)
+
+
+# Series 3
+print("---------------------------------")
+series_3 = ['Apples', 'Pears', 'Oranges', 'Peaches']
+
+# While loop to check for proper input and
+# remove fruit from list based on user tastes
+gross_fruit = []
+for f in series_3:
```

Comment:
If you loop through a copy like this: `for f in series_3[:]`, you can remove fruits from the original list inside your `for` loop without needing to build and loop through the `gross_fruit` list